long-lived-testnets/v2-052-1/README.md

```diff
@@ -0,0 +1,31 @@
+
+# `v2-052-1` Chain Details
+
+* **Chain-ID**: `v2-052-1`
+* **denom**: `stake`
+* **Application**: [`Simapp - Cosmos-SDK v0.50.x`](https://github.com/cosmos/cosmos-sdk/tree/release/v0.50.x/simapp)
```

Comment:
copy/pasty mistake, should be pointing to simapp v2 in release/v0.52

Reply:
fixed, thanks !